CHANGELOG.md

```diff
@@ -3,6 +3,67 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.28.2
+
+_Mar 28, 2025_
+
+We'd like to offer a big thanks to the 2 contributors who made this release possible. Here are some highlights ✨:
+
+- 🐞 Bugfixes
+
+Following are all team members who have contributed to this release:
+@flaviendelangle.
+
+<!--/ HIGHLIGHT_ABOVE_SEPARATOR /-->
+
+### Data Grid
+
+#### `@mui/x-data-grid@7.28.2`
+
+- [DataGrid] Fix error caused by trying to render rows that are not in the state anymore (#17117) @arminmeh
+
+#### `@mui/x-data-grid-pro@7.28.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-data-grid@7.28.2`.
+
+#### `@mui/x-data-grid-premium@7.28.2` [![premium](https://mui.com/r/x-premium-svg)](https://mui.com/r/x-premium-svg-link 'Premium plan')
+
+Same changes as in `@mui/x-data-grid-pro@7.28.2`.
+
+### Date and Time Pickers
+
+#### `@mui/x-date-pickers@7.28.2`
+
+- [fields] Improve the check for year in `doesSectionFormatHaveLeadingZeros` (#17112) @flaviendelangle
+
+#### `@mui/x-date-pickers-pro@7.28.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-date-pickers@7.28.2`.
+
+### Charts
+
+#### `@mui/x-charts@7.28.2`
+
+Internal changes.
+
+#### `@mui/x-charts-pro@7.28.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-charts@7.28.2`.
+
+### Tree View
+
+#### `@mui/x-tree-view@7.28.2`
+
+Internal changes.
+
+#### `@mui/x-tree-view-pro@7.28.2` [![pro](https://mui.com/r/x-pro-svg)](https://mui.com/r/x-pro-svg-link 'Pro plan')
+
+Same changes as in `@mui/x-tree-view@7.28.2`.
+
```

Comment:
Am I missing something?
But it looks like these packages did not have any changes to them. 🤔 🤷 
```suggestion
```

Reply:
No, you are correct, but previous releases kept them in the changelog. I can remove them if we all agree

Reply:
Are you sure?
If think we add entries when there are core/infra (non-functional) changes affecting a package (i.e. `x-internals` package changes, dep updates, etc.)

Reply:
I checked the changelog for 7.28.1 and I saw 'internal changes' as well, and the version was bumped. I assume this is when there are no changes to the package.